### PR TITLE
Kubeflow 1.7- manifest-prod

### PR DESCRIPTION
**Pull request by Souheil-Yazji**
Related to https://github.com/StatCan/aaw/issues/1793
But does not close it until we know all the patches are also included in. 

Will need to verify each  file makes sense

### CODE REVIEW

File: kustomize/contrib/spark/base/kustomization.yaml
Comment (wg102) [inline kustomize/contrib/spark/base/kustomization.yaml:1]: The upgrade was 1.7, but there was apparently also an upgrade to delete the file

Comment (wg102) [reply kustomize/contrib/spark/base/kustomization.yaml:1]: Jose Removed Spark 3 weeks ago  see https://github.com/StatCan/aaw-kubeflow-manifests/pull/373

File: kustomize/apps/centraldashboard/base/centraldashboard-config.yaml [hunk 1]
```diff
@@ -15,21 +15,27 @@ data:
           "link": "/en/",
           "text": "Notebooks",
           "icon": "book"
+        },
+        {
```
Comment (mathis-marcotte) [inline kustomize/apps/centraldashboard/base/centraldashboard-config.yaml:19]: S3 proxy is not ready for prod. So this menu link should not be put in

File: kustomize/apps/centraldashboard/base/centraldashboard-config.yaml [hunk 2]
```diff
@@ -56,21 +62,27 @@ data:
           "link": "/fr/",
           "text": "Serveur Bloc-notes",
           "icon": "book"
+        },
+        {
```
Comment (mathis-marcotte) [inline kustomize/apps/centraldashboard/base/centraldashboard-config.yaml:66]: Again, s3 proxy link

File: kustomize/apps/centraldashboard/base/centraldashboard-config.yaml (hunk 1 above)
Comment (mathis-marcotte) [inline kustomize/apps/centraldashboard/base/centraldashboard-config.yaml]: S3 proxy is not ready for prod so this link should be here yet.

File: kustomize/apps/training-operator/base/deployment.yaml [hunk 3]
```diff
@@ -9,7 +9,7 @@ spec:
       containers:
         - command:
             - /manager
-            - --mpi-kubectl-delivery-image=k8scc01covidacr.azurecr.io/mpioperator/kubectl-delivery:latest
+            - --mpi-kubectl-delivery-image=k8scc01covidacrdev.azurecr.io/mpioperator/kubectl-delivery:latest
```
Comment (Jose-Matsuda) [inline kustomize/apps/training-operator/base/deployment.yaml]: not sure if this should be using dev acr

File: kustomize/apps/centraldashboard/base/centraldashboard-config.yaml (hunk 2 above)
Comment (mathis-marcotte) [inline kustomize/apps/centraldashboard/base/centraldashboard-config.yaml]: Again, s3 proxy link

**Review by mathis-marcotte — CHANGES_REQUESTED** [review]

**Review by Jose-Matsuda — CHANGES_REQUESTED** [review]

File: kustomize/apps/training-operator/base/deployment.yaml (hunk 3 above)
Comment (wg102) [reply kustomize/apps/training-operator/base/deployment.yaml]: Updated

File: kustomize/apps/centraldashboard/base/centraldashboard-config.yaml (hunk 1 above)
Comment (wg102) [reply kustomize/apps/centraldashboard/base/centraldashboard-config.yaml]: Removed

File: kustomize/apps/centraldashboard/base/centraldashboard-config.yaml (hunk 2 above)
Comment (wg102) [reply kustomize/apps/centraldashboard/base/centraldashboard-config.yaml]: Removed

**Review by Jose-Matsuda — APPROVED** [review]

**Review by mathis-marcotte — APPROVED** [review]
lgtm

**Pull request merged**